### PR TITLE
[Doc] Add missing docs for the flink version 1.2.8

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/loading/Flink-connector-starrocks.md [hunk 1]
```diff
@@ -92,9 +93,9 @@ In your Maven project's `pom.xml` file, add the Flink connector as a dependency
...
+| sink.version                      | No           | AUTO              | The interface used to load data. This parameter is supported from Flink connector version 1.2.4 onwards. <ul><li>`V1`: Use [Stream Load](../loading/StreamLoad.md) interface to load data. Connectors before 1.2.4 only support this mode. </li> <li>`V2`: Use [Stream Load transaction](../loading/Stream_Load_transaction_interface.md) interface to load data. It requires StarRocks to be at least version 2.4. Recommends `V2` because it optimizes the memory usage and provides a more stable exactly-once implementation. </li> <li>`AUTO`: If the version of StarRocks supports transaction Stream Load, will choose `V2` automatically, otherwise choose `V1` </li></ul> |
```
Comment (amber-create) [inline docs/loading/Flink-connector-starrocks.md:97]: image
-> `V2` is recommended

Comment (amber-create) [inline docs/loading/Flink-connector-starrocks.md:97]: image
-> Stream Load transaction interface

Comment (amber-create) [inline docs/loading/Flink-connector-starrocks.md:97]: image
"who" will? This sentence needs a subject

Comment (amber-create) [inline docs/loading/Flink-connector-starrocks.md:97]: image
Always end a sentence with proper punctuation, for this sentence, use a period (.).

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-3.1 branch-3.0 branch-2.5

**Comment by hellolilyliuyi** [comment]
@mergify backport  branch-3.2

**Pull request merged**